recipes/fast-cdr/all/conandata.yml

```
@@ -20,3 +23,8 @@ sources:
   "1.0.21":
     url: "https://github.com/eProsima/Fast-CDR/archive/refs/tags/v1.0.21.tar.gz"
     sha256: "C1F32BDD76910ADA00D551EB8828DE7561AD2B2846D063CB4316F9262C03C77D"
+patches:
+  "2.0.0":
+    - patch_file: "patches/2.0.0-0001-Fix-for-non-CWG-1270-revision-compliant-compilers-17.patch"
+      patch_type: "conan"
+      patch_description: "Fix for non CWG 1270 revision compliant compilers"
```

```suggestion
      patch_type: "portability"
      patch_description: "Fix for non CWG 1270 revision compliant compilers"
      patch_source: "https://github.com/eProsima/Fast-CDR/pull/173"
```

Please, always add the origin of a new patch, so we can track in the future.

---

Makes sense to me, I have added the information as suggested in my last commit.

---

Oh, I just noticed that there was a button that would have simplified things...